### DISCUSSION TITLE
Create a new task

### DISCUSSION THREAD

**Discussion by HugoHeneault** [gwleuverink/clickup-time-tracker#32] [Ideas]
Coming from Paymo, we loved the fact that we could create tasks "on the go" by selecting first a project then a task, or create a task when it didn't exist. 

<img width="412" alt="image" src="https://user-images.githubusercontent.com/4112256/188647707-fc4e25bc-b911-4d9b-bb32-4541e9615ebe.png">

We could then even add them to tasks list :

<img width="478" alt="image" src="https://user-images.githubusercontent.com/4112256/188647808-6f3fc144-afc9-45aa-aec0-3af64894a85b.png">

This would be really nice to have ;)

**Comment by gwleuverink** [comment]
That is a great idea. Thanks for the suggestion. That would greatly reduce context switching.

If I'm not mistaking ClickUp makes it possible to configure required fields custom fields when creating a new task. I'd have to do some experimenting to see if this is straightforward to add.
